elasticsearch/values.yaml

```diff
@@ -71,7 +71,6 @@ networkHost: "0.0.0.0"
 
 volumeClaimTemplate:
   accessModes: [ "ReadWriteOnce" ]
-  storageClassName: "standard"
```

Comment:
Awesome! This is much better. Can you also update the defaults in the readme to match https://github.com/elastic/helm-charts/tree/master/elasticsearch#configuration

Reply:
updated readme